modules/machineset-creating-azure-ultra-disk.adoc

```diff
@@ -147,24 +148,70 @@ ifdef::mapi[]
             managedDisk:
               storageAccountType: UltraSSD_LRS
           userDataSecret:
-            name: worker-user-data-x5 <3>
+            name: <role>-user-data-x5 <3>
 endif::mapi[]
-             ...
 ----
-+
-<1> Specify a label to use to select a node that is created by this compute machine set. This procedure uses `disk.ultrassd` for this value.
+<1> Specify a label to use to select a node that is created by this machine set. This procedure uses `disk.ultrassd` for this value.
 <2> These lines enable the use of ultra disks.
 ifdef::mapi[]
 For `dataDisks`, include the entire stanza.
-<3> Specify the user data secret created earlier.
+<3> Specify the user data secret created earlier. Replace `<role>` with `{machine-role}`.
 endif::mapi[]
 
-. Create a compute machine set using the updated configuration by running the following command:
+. Create a machine set using the updated configuration by running the following command:
 +
 [source,terminal]
 ----
 $ oc create -f <machine-set-name>.yaml
 ----
+endif::cpmso[]
+
+ifdef::cpmso[]
+. Edit your control plane machine set CR by running the following command:
++
+[source,terminal]
+----
+$ oc --namespace openshift-machine-api edit controlplanemachineset.machine.openshift.io cluster
+----
+
+. Add the following lines in the positions indicated:
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: ControlPlaneMachineSet
+spec:
+  template:
+    spec:
+      metadata:
+        labels:
+          disk: ultrassd <1>
+      providerSpec:
+        value:
+          ultraSSDCapability: Enabled <2>
+          dataDisks: <2>
+          - nameSuffix: ultrassd
+            lun: 0
+            diskSizeGB: 4
+            deletionPolicy: Delete
+            cachingType: None
+            managedDisk:
+              storageAccountType: UltraSSD_LRS
+          userDataSecret:
+            name: <role>-user-data-x5 <3>
+----
+<1> Specify a label to use to select a node that is created by this machine set. This procedure uses `disk.ultrassd` for this value.
+<2> These lines enable the use of ultra disks. For `dataDisks`, include the entire stanza.
+<3> Specify the user data secret created earlier. Replace `<role>` with `{machine-role}`.
+
+. Save your changes.
+
+.Next steps
```

Comment:
@damdo said "Since we are mounting the Ultra disk on a specific path on the control plane machine, [users] need to reconfigure their workloads [...] to use this path. A reminder for this should suffice I think :)"

Not sure exactly what to add here for that, but I am guessing it would apply after the update is applied?

Reply:
Made an attempt! See "Next steps" in the refreshed preview.